client/view/Base/colors.css

```diff
@@ -12,8 +12,8 @@
   --text-underline-hover: oklch(23% 0 0);
   --accent-alternave: oklch(88% 0.16 92);
   --separator: oklch(23% 0 0 / 12%);
-  --warning: oklch(93.97% 0.094 94.62);
-  --error: oklch(70.31% 0.182 51.51 / 14%);
+  --warning: oklch(94% 0.09 95);
```

Comment:
Don’t forget to round OKLCH numbers

Reply:
I will be aware of this practice. Thanks!

Reply:
offtop:
![image](https://user-images.githubusercontent.com/22644149/189368042-a5a70fff-ecb5-4031-be8b-226f1cb8f10e.png)

Don't you want to suggest this practice in OKLCH Color Picker?

Reply:
Or is it unnecessary and too simple?)

Reply:
It will be nice, but a tricky thing